Document-Processing/Excel/Excel-Library/NET/faqs/how-to-change-data-point-label-color-of-a-waterfall-chart.md

```diff
@@ -74,7 +74,7 @@ End Using
 
 ## See Also
 
-* [How to change the grid line color of the Excel sheet?](https://help.syncfusion.com/file-formats/xlsio/faqs/how-to-change-the-grid-line-color-of-the-excel-sheet)
+* [How to change the grid line color of the Excel sheet?](https://help.syncfusion.com/document-processing/excel/excel-library/net/faqs/how-to-change-the-grid-line-color-of-the-excel-sheet)
 * [How to import data table with its data type using template markers?](https://help.syncfusion.com/file-formats/xlsio/faqs/how-to-import-data-table-with-its-data-type-using-template-markers)
```

Comment:
Please check and change the above link since it's redirecting. Check in all places